scripts/onnx/test.sh

```diff
@@ -70,4 +70,6 @@ if [[ "$BUILD_ENVIRONMENT" == *ort_test2* ]]; then
     pytest "${args[@]}" \
       "$top_dir/test/onnx/test_pytorch_onnx_onnxruntime.py::TestONNXRuntime_opset$i"
   done
+  pytest "${args[@]}" \
+    "$top_dir/test/onnx/test_pytorch_onnx_onnxruntime.py::TestONNXRuntime_opset12_onnx_shape_inference"
```

Comment:
My suggestion would be create a new file for the shape inference tests. `test_pytorch_onnx_onnxruntime.py` is already loaded with large number of test points, and pulling these into a separate file would be more organized.

Reply:
I see that you already have a new file `test/onnx/test_pytorch_onnx_shape_inference.py`, so my comment above may not be applicable. Still a bit surprised to see this line here.

Reply:
yes, putting it in `test_pytorch_onnx_onnxruntime.py` for one this is a test that runs ort, and then this is temporary and will be removed(or actually, replacing current tests) when shape inference is turned on by default.

Reply:
OK.